src/__tests__/renderer.test.tsx

```diff
@@ -763,4 +764,23 @@ describe('slack jsx', () => {
     );
     expect(await render(message)).toMatchSnapshot();
   });
+
+  it('renders an overflow menu', async () => {
+    const message = (
+      <OverflowMenu
```

Comment:
hmm can't exactly tell from usage, but an overflow menu element is actually a block, correct?

maybe it should be named drop-down menu block or menu block?

Reply:
it isn't a block, it is a block element. it could be a `OverflowMenuElement`

Reply:
perfect

Reply:
probably the unit test should show it in context too